src/amqp-client.cr

```diff
@@ -98,9 +103,14 @@ class AMQP::Client
     property nodelay, keepalive_idle, keepalive_interval, keepalive_count, send_buffer_size, recv_buffer_size
   end
 
+  record ConnectionInformation, name : String? = nil, product : String? = "amqp-client.cr", product_version : String? = nil, platform : String? = "Crystal", platform_version : String? = nil do
+    property name, product, product_version, platform, platform_version
+  end
+
   def initialize(@host = AMQP_HOST, @port = AMQP_PORT, @vhost = AMQP_VHOST, @user = AMQP_USER, @password = AMQP_PASS,
                  tls : TLSContext = AMQP_TLS, @websocket = AMQP_WS, @channel_max = 1024_u16, @frame_max = 131_072_u32, @heartbeat = 0_u16,
                  verify_mode = OpenSSL::SSL::VerifyMode::PEER, @name : String? = File.basename(PROGRAM_NAME),
+                 @connection_information = ConnectionInformation.new(File.basename(PROGRAM_NAME), "amqp-client.cr", AMQP::Client::VERSION, "Crystal", Crystal::VERSION),
```

Comment:
for backward compatibility this should be the last argument

Reply:
Yes!